app/css/index.css

```
@@ -173,15 +188,17 @@ button:disabled, button[disabled] {
 
 .check {
 	-webkit-appearance: none; /*hides the default checkbox*/
-	height: 20px;
-	width: 20px;
+	height: 35px;
+	width: 35px;
 	position: relative;
 	background-color: #FE0006;
 	text-align: center;
 	font-weight: 600;
 	color: white;
 	border-radius: 3px;
 	outline: none;
+	font-size: x-large;
+	align-items: center
```

`align-items` only applies to grid or flex items, missing semi-colon and the `display: grid;` declaration

```suggestion
	align-items: center;
	display: grid;
```